mobile/android/base/android_strings.dtd

```diff
@@ -16,7 +16,7 @@
 <!ENTITY firstrun_bookmarks_subtext "Wyświetlaj wyniki z zakładek i historii podczas wyszukiwania.">
 <!ENTITY firstrun_data_title "Dane">
 <!ENTITY firstrun_data_message "Mniej danych, więcej oszczędności">
-<!ENTITY firstrun_data_subtext2 "Wyłącz obrazy, aby zmniejszyć zużycie danych.">
+<!ENTITY firstrun_data_subtext2 "Wyłącz obrazki, aby zmniejszyć zużycie danych.">
```

Comment:
Personally, I wouldn't be against changing this to normal form I think - maybe [report an issue](https://github.com/aviarypl/team/issues) for this?